index.html

```diff
@@ -196,40 +207,54 @@ <h2>About</h2>
 <!-- foooter -->
 <footer class="contact_section">
     <div class="mobile_logo"></div>
+
+    <div class="footer_div">
+
+
     
-      <h4 class="title-contact text_header_font">Get Started</h4>
-
-      <p class="section-desc text_body_font">
-        Hello!I am a software developer! I can 
-        help you build a product, feature or website.
-        Take a look of my works. If you like what you see 
-        and have a project you need coded, don't hesitate 
-        and contact me.
-      </p>
-      <form action="https://formspree.io/f/mzbovllg" method="post">
-          <input type="text" name="FullName" placeholder="Enter Your Name" maxlength="30" required>
-          <input type="email" name="Email" placeholder="Enter Your Email" required>
-          <textarea name="Message" placeholder="Enter Your Message"  maxlength="500" required></textarea>
-          <button class="" type="submit" name="submit-btn">Start collaboration</button>
-      </form>
-
-      <div class="bottom_row">
-        <a class="text-button text_body_font resume_color">
-            <p>Get my Resume</p>
-            <div class="arrow_upload"></div>
-        </a>
-
-        <div class="divicon">
-            <ul class="sociallist">
-           <li class="sociallist_link"><a href="#"><img src="img/twitter.png" alt="twitter"></a></li>
-           <li class="sociallist_link" ><a href="#"><img src="img/linkdin.png" alt="linkdin"></a></li>
-           <li class="sociallist_link" ><a href="#"><img src="img/medium.png" alt="medium"></a></li>
-           <li class="sociallist_link"><a href="#"><img src="img/github.png" alt="github"></a></li>
-           
-            </ul>
+    
+        <div class="contact_div">
+
+    
+            <h4 class="title-contact text_header_font">Get Started</h4>
+
+            <p class="section-desc text_body_font">
+                Hello!I am a software developer! I can 
+                help you build a product, feature or website.
+                Take a look of my works. If you like what you see 
+                and have a project you need coded, don't hesitate 
+                and contact me.
+            </p>
+            <form action="https://formspree.io/f/mzbovllg" method="post">
+                <input type="text" name="FullName" placeholder="Enter Your Name" maxlength="30" required>
+                <input type="email" name="Email" placeholder="Enter Your Email" required>
+                <textarea name="Message" placeholder="Enter Your Message"  maxlength="500" required></textarea>
+                <button class="" type="submit" name="submit-btn">Start collaboration</button>

```

Comment:
- [ ] Kindly your `contact` section overflows, kindly make it fit the page width.
- [ ] Kindly there are 2 background images missing kindly add them.

Your design
![image](https://user-images.githubusercontent.com/35535130/160385510-23420ff4-7c92-426a-8081-2388d4e48e62.png)

Figma design
![image](https://user-images.githubusercontent.com/35535130/160385621-5a276776-980c-47e6-9245-448701fdc029.png)